backend/src/sockets/disconnect.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
not sure if we need a disconnect listener, its probably better to emit an event, and disconnect from the client side (e.g. from endGame method in the Game class)

Reply:
I think we need one. If a user closes the tab or "disconnects" some how we need to pick up on this event and delete the user from the Game.Session object. Or else when the user connects again, we'll get null errors.

Reply:
true